talpid-core/Cargo.toml

```diff
@@ -31,7 +31,7 @@ rand = "0.7"
 
 
 [target.'cfg(not(target_os="android"))'.dependencies]
-openvpn-plugin = { git = "https://github.com/mullvad/openvpn-plugin-rs", branch = "auth-failed-event", features = ["serde"] }
+openvpn-plugin = { git = "https://github.com/mullvad/openvpn-plugin-rs", rev = "d33b0d2795b99c83804d4c757acef881b901a5d0", features = ["serde", "auth-failed-event"] }
```

Comment:
This is temporary. I will publish `0.4.0` and depend on that once this PR has been approved. I wanted this code change to get approved before I went ahead and actually published anything.